apiserver/server_test.go

```diff
@@ -53,6 +54,12 @@ type serverSuite struct {
 
 var _ = gc.Suite(&serverSuite{})
 
+func (s *serverSuite) SetUpSuite(c *gc.C) {
+	s.JujuConnSuite.SetUpSuite(c)
+	s.PatchValue(&corecert.NewCA, coretesting.NewCA)
```

Comment:
Neat - I hadn't really clicked that PatchValue was clever enough to leave patches done in the suite setup until suite teardown.